regression-test/suites/cloud_p0/auth/test_grant_revoke_cluster_to_user.groovy

```diff
@@ -98,7 +98,7 @@ suite("test_grant_revoke_cluster_to_user", "cloud_auth") {
     sql """create user ${user3} identified by 'Cloud12345'"""
     sql """GRANT SELECT_PRIV ON *.*.* TO '${user3}'@'%'"""
     result = connect(user = "${user3}", password = 'Cloud12345', url = context.config.jdbcUrl) {
-            sql """SHOW CLUSTERS"""
+            sql """SHOW COMPUTE GROUPS"""
```

Comment:
Maybe we should keep both `SHOW CLUSTERS` and `SHOW COMPUTE GROUPS`, otherwise, either of them may be not tested.